src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -88,30 +95,17 @@ export function CreateExportModal({
       >
         <Group c="dimmed" mb="xs">
           <Text>{project?.name}</Text>
-          <Tooltip
-            withArrow
-            multiline
-            w={150}
-            fz={14}
-            fw={500}
-            openDelay={200}
-            closeDelay={200}
-            ta="center"
-            color="primaryBlue"
-            variant="filled"
-            label="Only issues with corresponding types and a 'Done' status are exported. The remaining status influence the date calculations."
-          >
-            <ActionIcon variant="subtle" ml="auto">
-              <IconInfoCircle />
-            </ActionIcon>
-          </Tooltip>
+          <CloseButton ml="auto" onClick={() => handleClose()} />
```

Comment:
```suggestion
          <CloseButton ml="auto" onClick={handleClose} />
```
Can shorten this

Reply:
sadly, eslint does not allow me to do this :(